mlserver/cli/main.py

```diff
@@ -47,13 +47,17 @@ async def start(folder: str):
 @root.command("build")
 @click.argument("folder", nargs=1)
 @click.option("-t", "--tag", type=str)
+@click.option("--cache/--no-cache", default=False)
 @click_async
-async def build(folder: str, tag: str):
+async def build(folder: str, tag: str, cache: bool):
     """
     Build a Docker image for a custom MLServer runtime.
     """
     dockerfile = generate_dockerfile()
-    build_image(folder, dockerfile, tag)
+    if cache:
+        build_image(folder, dockerfile, tag)
+    else:
+        build_image(folder, dockerfile, tag, no_cache=True)
```

Comment:
nit: should we just change the `no_cache` kwarg to `cache=cache`? That way we could get rid of this `if` / `else`.